backend/hardware/grbl/core.py

```diff
@@ -0,0 +1,15 @@
+"""
+This module contains the implementations for grbl control. See base.py for the abstract class used
+and the individual files for configuration information.
+"""
+
+
+def createGRBL(grblConfig: dict, devices: dict):
```

Comment:
It looks like `devices` isn't being used in this function

Reply:
It is currently unused because neither of the grbl implementations need to reference other devices, but I would prefer to keep it there just so the function is consistent with the other "createX" functions, and since future grbl implementations or changes to the current implementation might need it. I changed it to _devices just to mark it as currently unused though.